recipes/orange3/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "3.3.6" %}
+
+package:
+  name: orange3
```

Comment:
Normally, I would want to call this just `orange` and let be versioned. However, this is the name they chose for PyPI too.